src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -214,6 +206,19 @@ Here's the complete example from start to finish. This example assumes the Infus
 
         <nav class="flc-toc-tocContainer"> </nav>
 
+        <!-- Initialize the UI Options Javascript -->
+        <script type="text/javascript">
+            $(document).ready(function () {
+                fluid.uiOptions.prefsEditor(".flc-prefsEditor-separatedPanel", {
+                    tocTemplate: "lib/infusion/src/components/tableOfContents/html/TableOfContents.html",
+                    terms: {
+                        templatePrefix: "lib/infusion/src/framework/preferences/html",
+                        messagePrefix: "lib/infusion/src/framework/preferences/messages"
+                    }
+                });
+            })
+        </script>
```

Comment:
@jobara I have no opinion. I recommend you ask other people for their input. All I know is that this works when I run it. How good it is you'll have to ask someone else.